src/components/common/FormikUICustomMultipleSelect.tsx

```diff
@@ -13,6 +13,7 @@ const FormikUICustomMultipleSelect = ({
   availableOptions,
   id,
   label,
+  width,
```

Comment:
Maybe param should be optional as it has the default value

Reply:
I think I solved this in the previous PR https://github.com/UserOfficeProject/user-office-frontend/pull/49. And the changes are not here because I haven't pulled from thereafter I solved that. That is why I mentioned that this should be merged after https://github.com/UserOfficeProject/user-office-frontend/pull/49